fe/fe-core/src/main/java/com/starrocks/clone/TabletSchedCtx.java

```diff
@@ -906,11 +908,11 @@ public CreateReplicaTask createEmptyReplicaAndTask() throws SchedException {
                     olapTable.getPartitionInfo().getTabletType(partitionId),
                     olapTable.getCompressionType(), indexMeta.getSortKeyIdxes(),
                     indexMeta.getSortKeyUniqueIds());
-            createReplicaTask.setIsRecoverTask(true);
+            createReplicaTask.setIsFromScheduler(true);
```

Comment:
suggest:
setIsFromScheduler -> setIsRecoveryFromScheduler
setIsRecoverTask -> setIsRecoveryFromReportHandler

ugh... looks ugly, how about add a new state to repalce those two: Enum recoverySrc = {SCHEDULER, REPROT, NONE}